### DISCUSSION TITLE
Submitted a pull request is there anything else i have to do later ?

### DISCUSSION THREAD

**Discussion by AmeyaGawde326** [Docile-Alligator/Infinity-For-Reddit#1146] [General]
Actually I added a new feature to open multiple reddit tabs on top of each other it was 10 days ago is there a reason why it wasn't merged is there something else i have to do 

I am new to opensource and coding in general so if there's something please let me know

**Comment by Wladefant** [comment]
We are checking and thinking about your pull request. The next time Before submitting a pull request with a completely  new feature contat us.

**Reply by AmeyaGawde326** [reply]
Thanks for clarifying
Really appreciate it 

There's already an reddit app called reddit sync which has the same feature and i really liked so so i reverse engineered it here 

Next time I'll be sure to inform first 
👍
